hubmap/hubmap/urls.py

```diff
@@ -13,11 +13,41 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from django.conf import settings
+from django.conf.urls import url
 from django.contrib import admin
 from django.urls import include, path
+from django.views.generic import TemplateView
+from rest_framework.authtoken import views
+from rest_framework.documentation import include_docs_urls
+from rest_framework.schemas import get_schema_view
+
+from dataportal import views as dataportal_views
```

Comment:
It would be preferable not to have an absolute import here; PyCharm doesn't understand this and a relative import (`from ..dataportal import ...`) fails because the upper-level directory isn't a Python package. It might be appropriate to add an empty `__init__.py` file to the parent directory, but I'm not sure.